docs/content/en/docs/tutorials/tetragon-metrics.md

```diff
@@ -73,6 +73,40 @@ promhttp_metric_handler_errors_total{cause="gathering"} 0
 ...
 ```
 
-### Container
+## Container
```

Comment:
For container please let's @Vampouille contribute this part in https://github.com/cilium/tetragon/pull/1503/  ;-)

Reply:
Sure !